src/box/memtx_engine.cc

```diff
@@ -1371,6 +1371,7 @@ memtx_engine_memory_stat(struct engine *engine, struct engine_memory_stat *stat)
 
 static const struct engine_vtab memtx_engine_vtab = {
 	/* .free = */ memtx_engine_free,
+	/* .shutdown = */ NULL,
```

Comment:
The new engine method should be introduced in the patch renaming `shutdown` to `free` or in a separate patch.

Reply:
Done in the renaming patch.